provider/internal/java/dependency.go

```diff
@@ -204,30 +203,34 @@ func (p *javaServiceClient) GetDependenciesDAG(ctx context.Context) (map[uri.URI
 	if p.mvnSettingsFile != "" {
 		args = append(args, "-s", p.mvnSettingsFile)
 	}
+
 	// get the graph output
 	cmd := exec.Command("mvn", args...)
 	cmd.Dir = moddir
-	err = cmd.Run()
+	mvnOutput, err := cmd.CombinedOutput()
 	if err != nil {
 		return nil, err
 	}
 
+	if _, err = f.Write(mvnOutput); err != nil {
```

Comment:
I don't know why we would write and then read from the file for it to be removed. I feel like remove everything to do with the file.